doc(translate): Additional Chinese translation

iamxiaojianzheng opened the pull request:
- **PR Description**

- **Please check if the PR fulfills these requirements**

* [x] Cheatsheets are up-to-date (run `go generate ./...`)
* [x] Code has been formatted (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#code-formatting))
* [ ] Tests have been added/updated (see [here](https://github.com/jesseduffield/lazygit/blob/master/pkg/integration/README.md) for the integration test guide)
* [x] Text is internationalised (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#internationalisation))
* [ ] Docs have been updated if necessary
* [x] You've read through your own file changes for silly mistakes etc

<!--
Be sure to name your PR with an imperative e.g. 'Add worktrees view'
see https://github.com/jesseduffield/lazygit/releases/tag/v0.40.0 for examples
-->

stefanhaller commented:
@iamxiaojianzheng Thanks for the contribution! Unfortunately we can't use it, as the JSON file that you changed is machine-generated and shouldn't be edited manually. I just added a README file to the directory containing these files to make this clearer, hopefully (#3781).

We recently had another PR (#3766) that also changed the same file; in that case I took the JSON file and uploaded it to crowdin, so that the changes are not lost. I don't want to do the same thing again with your PR, as I'm worried about overwriting the changes from that other PR. This is not the workflow we want to use; please get an account at Crowdin and edit the translations there.

iamxiaojianzheng commented:
> @iamxiaojianzheng Thanks for the contribution! Unfortunately we can't use it, as the JSON file that you changed is machine-generated and shouldn't be edited manually. I just added a README file to the directory containing these files to make this clearer, hopefully (#3781).
> 
> We recently had another PR (#3766) that also changed the same file; in that case I took the JSON file and uploaded it to crowdin, so that the changes are not lost. I don't want to do the same thing again with your PR, as I'm worried about overwriting the changes from that other PR. This is not the workflow we want to use; please get an account at Crowdin and edit the translations there.

Ok，I have completed most of the translations and hope to help those in need.

stefanhaller commented:
@iamxiaojianzheng Thank you very much! These will show up in lazygit the next time we update the JSON files from Crowdin, which will happen before the next release at the latest.
